recipe/meta.yaml

```
@@ -19,15 +19,17 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8,<3.10
     - pip
   run:
-    - python >=3.7
+    - python >=3.8,<3.10
     - dask >=2021.11.1,<=2022.01.0
     - distributed >=2021.11.1,<=2022.01.0
```

Think we need to [bump these]( https://github.com/rapidsai/dask-cuda/blob/451b3b3a5fb5aa2a194d8acbb966ccaaa020fff1/requirements.txt#L1-L2 ) too

```suggestion
    - dask==2022.03.0
    - distributed==2022.03.0
```